IntBuilder.js

```diff
@@ -0,0 +1,66 @@
+import BuilderES6 from './BuilderES6.js'
+
+// ES6 Way
+export default class IntBuilder extends BuilderES6 {
+  constructor (operand) {
+    super(operand ?? 0)
+    IntBuilder.#throwErrorIfValueNotAnInteger(this._currentResult)
+    this._currentResult = +this._currentResult
```

Comment:
Why do you need this line?

Reply:
You're right, I don't need it. It just that I firstly decided to allow strings to be converted then to number, but then changed my mind and implemented method that throws error if value not an integer. So this line makes no sense now.

Reply:
remove the unnecessary conversion